pkgs/tools/misc/migra/default.nix

```
@@ -0,0 +1,41 @@
+{
+  lib,
+  python3,
+  fetchPypi,
+}: let
+  pname = "migra";
+  version = "3.0.1647431138";
+in
+  python3.pkgs.buildPythonApplication rec {
+    inherit pname version;
```

Is there a reason you use the `let ... in` pattern here? You also use `rec`...

```suggestion
}:

python3.pkgs.buildPythonApplication rec {
  pname = "migra";
  version = "3.0.1647431138";
```